pkg/eventhandler/eventhandlers.go

```diff
@@ -62,6 +67,7 @@ type EventHandler struct {
 	ImageFilter     ImageFilter
 	Mapper          EventMapper
 	K8s             K8s
+	ErrorSender     UniformErrorLogSender // TODO initialize correctly
```

Comment:
what abou tthe Todo?

Reply:
It was done but I forgot to remove it :stuck_out_tongue_winking_eye: